test/integration/features/coolstore.feature

```diff
@@ -1,4 +1,4 @@
-@coolstore
+@coolstore @centos-only
```

Comment:
@hferentschik Idea was to not drive the the information about image by `--iso-url centos` but instead use environmental variable and prevent it from running when boot2docker is used.